base/twiceprecision.jl

```diff
@@ -789,3 +789,21 @@ _tp_prod(t::TwicePrecision) = t
     x.hi < y.hi || ((x.hi == y.hi) & (x.lo < y.lo))
 
 isbetween(a, x, b) = a <= x <= b || b <= x <= a
+
+# For NaN/Inf/negative this returns junk
+function _log_twice64_unchecked(x::Float64)
```

Comment:
note that this isn't full twice precision it's only about 70 bits while true twice precision would be 106. not sure that matters though.

Reply:
Are you thinking the comment should be scarier, or the name?

This seems like plenty of bits for this use. Could also be named / located near where it's used.